### ISSUE TITLE
固网用不了，手机4G网络可以用，何解？

### ISSUE THREAD

**Issue by Y86i3JX9** [shadowsocks/shadowsocks-windows#2287]
### Shadowsocks version / 影梭版本
服务器Centos端版本：shadowsocks-libev 3.2.0
Windows客户端版本：4.1.5

### Environment(Operating system, .NET Framework, etc) / 使用环境（操作系统，.NET Framework等）
Windows 10 64bit

### Steps you have tried / 操作步骤
固网是北京联通的，在电脑上无法翻墙，日志没有特殊提示，服务器的TCP连接都是正常的，换过端口，增加过插件（simple-obfs和GoQuiet）都没有解决问题，手机使用固网的WiFi也不能连。

但比较怪异的是，手机关掉固网WiFi，使用移动和联通的4G网络，就可以正常翻墙，没有任何问题。

该现象应该持续很长时间了，我有好几个月没回家了，所以不知道是什么时候开始这样的。

### What did you expect to see? / 期望的结果
电脑可以正常翻墙。

### What did you see instead? / 实际结果
电脑完全无法翻墙。

### Config and error log in detail (with all sensitive info masked) / 配置文件和日志文件（请隐去敏感信息）
以下是日志：

```
[2019-03-21 ] Shadowsocks started (4.1.5)
[2019-03-21 ] 
=========================
Registered Encryptor Info
aes-128-cfb=>StreamOpenSSLEncryptor
aes-192-cfb=>StreamOpenSSLEncryptor
aes-256-cfb=>StreamOpenSSLEncryptor
aes-128-ctr=>StreamOpenSSLEncryptor
aes-192-ctr=>StreamOpenSSLEncryptor
aes-256-ctr=>StreamOpenSSLEncryptor
bf-cfb=>StreamOpenSSLEncryptor
camellia-128-cfb=>StreamOpenSSLEncryptor
camellia-192-cfb=>StreamOpenSSLEncryptor
camellia-256-cfb=>StreamOpenSSLEncryptor
rc4-md5=>StreamOpenSSLEncryptor
salsa20=>StreamSodiumEncryptor
chacha20=>StreamSodiumEncryptor
chacha20-ietf=>StreamSodiumEncryptor
aes-128-gcm=>AEADOpenSSLEncryptor
aes-192-gcm=>AEADOpenSSLEncryptor
aes-256-gcm=>AEADOpenSSLEncryptor
chacha20-ietf-poly1305=>AEADOpenSSLEncryptor
xchacha20-ietf-poly1305=>AEADSodiumEncryptor
=========================
//时间也统一隐藏
[2019-03-21 00:00:00] connect to ajax.googleapis.com:443
[2019-03-21 00:00:00] Socket connected to ss server: 隐藏我的服务器地址:443
[2019-03-21 00:00:00] connect to ajax.googleapis.com:443
[2019-03-21 00:00:00] Socket connected to ss server: 隐藏我的服务器地址:443
[2019-03-21 00:00:00] connect to ajax.googleapis.com:443
[2019-03-21 00:00:00] Socket connected to ss server: 隐藏我的服务器地址:443
[2019-03-21 00:00:00] connect to ajax.googleapis.com:443
[2019-03-21 00:00:00] Socket connected to ss server: 隐藏我的服务器地址:443
[2019-03-21 00:00:00] connect to clients4.google.com:443
[2019-03-21 00:00:00] Socket connected to ss server: 隐藏我的服务器地址:443
[2019-03-21 00:00:00] connect to clients4.google.com:443
[2019-03-21 00:00:00] Socket connected to ss server: 隐藏我的服务器地址:443
```

**Comment by chenshaoju** [comment]
该问题怀疑和SS原生协议已经被检出有关，如果你的服务器是自己搭的，请尝试套一个插件看看：https://github.com/shadowsocks/shadowsocks-windows/issues/2228#issuecomment-465047836

**Comment by Y86i3JX9** [comment]
谢谢陈兄。问题是即便套上了插件，也无济于事。现在不知道还有没有其他方式实现翻墙？我自己的海外服务器可以正常访问，TCP之类的都是通的，但就是不能用SS了。

**Comment by chenshaoju** [comment]
套了插件以后可能得换一个端口。

如果仍然不行，可能得换IP了……

**Comment by celeron533** [comment]
另外如果是pppoe拨号，请把连接名称改为英文

**Comment by akirasen** [comment]
其它梯子 不谢 https://fileem.com/how-can-the-shadowsocket-ladder-not-be-used-2-5-knives-a-month-try-to-build-a-ladder-with-v2ray-ten-minutes-to-build-the-most-complete-ladder-tutorial-in-history

**Comment by 752021113** [comment]
玄学问题 同样节点4g可用WiFi不可用    手机热点可用     
解决办法：PC端全部节点删除重写手动输入。。。。解决。。。。
